### PR TITLE
Fix pronunciation of `conch` in readme

### PR DESCRIPTION
**Pull request by Carreau**
My phonetic is not perfect, and might carry a bit of my French accent,

Would like @scopatz review on that.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -17,7 +17,7 @@ xonsh
 
 xonsh is a Python-ish, BASHwards-looking shell language and command prompt.
 The language is a superset of Python 3.4+ with additional shell primitives.
-xonsh (pronounced *conch*) is meant for the daily use of experts and novices 
+xonsh (pronounced *🐚*) is meant for the daily use of experts and novices
```
Comment (scopatz) [inline README.rst]: I think this breaks the word play.

Comment (scopatz) [inline README.rst]: And this would prefer to keep it the way it is :).

**Comment by Carreau** [comment]
Ok, just added the shell in the docs.

**Comment by scopatz** [comment]
thanks!

**Pull request merged**